samples/compression/compress_common.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2015 Intel(r) Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http ://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "compress_common.hpp"
+
+#include <cmath>
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+using namespace vmf;
+
+const string GPS_DESC = "gps";
+const string GPS_COORD_LAT_FIELD = "lat";
+const string GPS_COORD_LNG_FIELD = "lng";
+const string GPS_SCHEMA_NAME = "gps_coords_schema";
+const double PI = 3.14159265358979323846;
+
+void writeMetadataStream(MetadataStream& mdStream, const vmf_string& videoFile)
```

Comment:
do you agree that it's better to call this function `generateMetadata()` and don't provide file to it?